habitat_sim/robots/fetch_robot.py

```diff
@@ -0,0 +1,159 @@
+import magnum as mn
+import numpy as np
+import pybullet as p
```

Comment:
Consider making this soft.  You can do a similar thing like was done with torch where you import once it's needed.